pkg/inject/proxy.go

```diff
@@ -56,18 +57,22 @@ type proxyConfig struct {
 	proxyIngressPort int64
 	// UID used by proxy
 	proxyUID int64
+	// todo
+	enableIPV6 *bool
```

Comment:
Does this need to be a *bool? Why not just a bool?

Reply:
I found when using a bool instead of *bool, and `enableIPV6` was not set explicitly in `proxyConfig`, that the default value was `false`. This resulted in unexpected value of `APPMESH_ENABLE_IPV6: 0` in the proxyinit env vars.

This allows checking for `m.proxyConfig.enableIPV6 != nil && !*m.proxyConfig.enableIPV6` ensuring that the default value of `APPMESH_ENABLE_IPV6` remains `1`.

Reply:
Ah understood. Can you add a comment explaining this? I want to make sure the next person who looks at the code understands this is to set the default to 1 when the variable is unset.